CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ Cloudant Sync - Android is written in Java and uses
 The main requirements are:
 
 * Java 1.6
-* Gradle 1.8+
+* Gradle 2.2.1+
```

Comment:
Is this still a requirement for most developers if they are instructed to use `gradlew`?